packages/twenty-front/src/modules/settings/data-model/fields/forms/boolean/hooks/useBooleanSettingsFormInitialValues.ts

```diff
@@ -0,0 +1,27 @@
+import { useFormContext } from 'react-hook-form';
+
+import { FieldMetadataItem } from '@/object-metadata/types/FieldMetadataItem';
+import { SettingsDataModelFieldBooleanFormValues } from '@/settings/data-model/fields/forms/boolean/components/SettingsDataModelFieldBooleanForm';
+
+export const useBooleanSettingsFormInitialValues = ({
+  fieldMetadataItem,
+}: {
+  fieldMetadataItem?: Pick<FieldMetadataItem, 'defaultValue'>;
+}) => {
+  const initialDefaultValue =
+    (fieldMetadataItem?.defaultValue as
+      | SettingsDataModelFieldBooleanFormValues['defaultValue']
+      | undefined) ?? true;
```

Comment:
why do we type it to possible undefined if we have ?? true anyway?

Reply:
Done!